public/app/features/canvas/elements/icon.tsx

```diff
@@ -101,7 +101,7 @@ export const iconItem: CanvasElementItem<IconConfig, IconData> = {
 
     const data: IconData = {
       path,
-      fill: cfg.fill ? ctx.getColor(cfg.fill).value() : '#CCC',
+      fill: cfg.fill ? ctx.getColor(cfg.fill).value() : '#19730E',
```

Comment:
:thinking: I think this only appears to fix things (does that mean it is fixed?!)

IIUC, the real issue is that the config is undefined when it starts so it falls back too a default value.  The UI does the same thing.  This changes makes them both default to the same (in dark mode) but I don't think it fixes the general issue that will keep coming up if we don't actually initialize the settings

Reply:
Ah yes, on looking at this again it looks like we were just missing including a fill property on the root icon object in `model.gen`